bcda/testUtils/client.go

```diff
@@ -15,7 +15,7 @@ type BlueButtonClient struct {
 	HICN *string
 }
 
-func (bbc *BlueButtonClient) GetExplanationOfBenefitData(patientID string, jobID string) (string, error) {
+func (bbc *BlueButtonClient) GetExplanationOfBenefitData(patientID, jobID, cmsID string) (string, error) {
 	args := bbc.Called(patientID)
```

Comment:
Should the parameters on this `.Called()` be updated also? (I think you mentioned this in our discussion earlier)

Reply:
yes they should and the GetData method in this class also. Problem is updating will cause failures in our main_test.go, and I started to feel like the change was out of scope of what this ticket should be doing. But I do think a ticket should be created to not only update this but also make our main_test.go more consistent in either having all of the endpoints tested or none, similar to our earlier convo in slack.